### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by hemisemidemipresent** [gurrenm3/BloonsTD6-Mod-Helper#2]
could there be some documentation on what function does what? It doesnt have to be fancy, just a simple .md file would help

**Comment by gurrenm3** [comment]
Yes, the new mod helper is already heavily documented within the code and I'll be making a .md file for it soon. This current repo is no longer supported, and instead you should use this one because it works for BATTD as well https://github.com/gurrenm3/BTD-Mod-Helper
